### PR TITLE
Feature/redesign mobile tablet

### PR DESCRIPTION
**Pull request by flipkoehler**

### CODE REVIEW

File: components/Benefits/Benefits.js [hunk 1]
```diff
@@ -4,62 +4,75 @@ import Image from "next/image";
...
+  background-color: var(--globalWhite);
+  width: 4rem;
+  height: 4rem;
+  border-radius: 50%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
 `;
+
+const StyledImage = styled(Image)``;
```
Comment (AntjeWalter) [inline components/Benefits/Benefits.js:78]: you could delete this, if it's not needed anymore :)

File: components/Slider/Slider.tsx [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+  title: string;
+  content: string;
+  buttonText: string;
+  isButtonDissabled: boolean;
+  url: string;
+}
+
+type SliderContentArrayType = {
+  sliderContent: SliderContent[];
+};
```
Comment (AntjeWalter) [inline components/Slider/Slider.tsx:18]: Okay, I have never seen this, therefore I can't really say anything about the code but it looks pretty cool! :D

File: components/WelcomePage/WelcomePage.tsx [hunk 3]
```diff
@@ -3,34 +3,60 @@
...
+          Du hast das ewige Scrollen durch Netflix, Amazon und Co. satt? Mit der
+          Hilfe von vier kurzen Fragen geben wir dir eine wunderbare
+          Filmempfehlung für deinen nächsten Filmabend. Alle Filme sind
+          handverlesen und garantieren beste Unterhaltung!
+          <StyledLink href="/quiz">Quiz jetzt starten!</StyledLink>
+        </StyledPIntroText>
+
+        <StyledDivLottie>
+          <Lottie animationData={movieCouchRelaxe} loop={true} />
+        </StyledDivLottie>
```
Comment (AntjeWalter) [inline components/WelcomePage/WelcomePage.tsx:29]: nice!!

**Review by AntjeWalter — APPROVED** [review]
Can't say much but looks awesome!

File: components/Slider/Slider.tsx (hunk 2 above)
Comment (flipkoehler) [reply components/Slider/Slider.tsx:18]: i can highly recommend the slider plugin :)

File: components/WelcomePage/WelcomePage.tsx (hunk 3 above)
Comment (flipkoehler) [reply components/WelcomePage/WelcomePage.tsx:29]: bling bling

**Pull request merged**